src/app/tables/table.component.html

```diff
@@ -83,6 +85,11 @@ <h2 class="table-title" *ngIf="title">
                   </ng-template>
                 </td>
               </ng-container>
+              <td *ngIf="nodeStoryUrls">
+                <a [href]="nodeStoryUrls[item[uidColumn]]" target="_blank">
```

Comment:
Even if `nodeStoryUrls` is `{}`, it will evaluate to `true`. For example, you can try `if ({}) {console.log("Hello")}` in a Node console. My suggestion is to also add `*ngIf="nodeStoryUrls[item[uidColumn]] !== undefined"` to the anchor element so it doesn't appear if there's no Story URL associated with the `item`.

Reply:
I tried this and its breaking tests, but I am not sure why. Will keep debugging, but let me know if you have ideas (see Travis for failure)

Reply:
I provided a suggestion above. The issue is that sometimes `nodeStoryUrls` is undefined, so you can't access a property when it's not an object.